pages/thanks-rider.html

```diff
@@ -30,4 +41,5 @@
             </div>
         </div>
     </div>
+    <script src="/scripts/thanks.js"></script>
```

Comment:
The bit I added to the front-matter was `thanks: true`. If you like I can handle it, as discussed above.